src/bundle/Resources/views/admin/bookmark/list.html.twig

```diff
@@ -115,6 +117,10 @@
     </div>
 {% endblock %}
 
+{% block react_modules %}
```

Comment:
I moved Content Tree container div to parent twig, but I've left JS in lower templates as I don't want to load them globally.

Reply:
If anyone has idea how to solve this problem I would readily hear it. :)